sys-process/glances/glances-2.7.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_{4,5}} )
+PYTHON_REQ_USE="ncurses"
+
+inherit distutils-r1 eutils linux-info
+
+MYPN=Glances
+MYP=${MYPN}-${PV}
```

Comment:
Instead of `${MYP}`, use `${P^}` in EAPI 6, and then get rid of all the `MYPN` and `MYP`'s